.circleci/config.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           type: instrumentation
           apk-path: WordPress/build/outputs/apk/vanilla/debug/WordPress-vanilla-debug.apk
           test-apk-path: WordPress/build/outputs/apk/androidTest/vanilla/debug/WordPress-vanilla-debug-androidTest.apk
-          test-targets: package org.wordpress.android.e2e
+          test-targets: notPackage org.wordpress.android.ui.screenshots
```

Comment:
Running everything except `org.wordpress.android.ui.screenshots` means we also run the existing non-UI connected tests.